src/scenes/start_scene.py

```diff
@@ -179,6 +185,17 @@ def load_game(self, game_id: int) -> None:
                 game_status = tree_root.find("level/phase").text.strip()
                 turn_nb = int(tree_root.find("level/turn").text.strip())
 
+                print("------------------------------------")
```

Comment:
Let's avoid pushing print statements to the repository, it's a bad practice.

I can give you more insights about why it's bad if you are wondering why.